src/Toolchain/DefaultToolchain.ts

```diff
@@ -32,49 +32,12 @@ class DefaultToolchain {
     return this._instance || (this._instance = new this());
   }
 
-  /* istanbul ignore next */
-  public async ask(
-    toolchainEnv: ToolchainEnv,
-    toolchain: Toolchain
-  ): Promise<boolean> {
-    return new Promise<boolean>((resolve, reject) => {
-      const name = `${toolchain.info.name}-${toolchain.info.version?.str()}`;
-      vscode.window
-        .showInformationMessage(
-          `Do you want to use ${name} as a default toolchain?`,
-          "Yes",
-          "No"
-        )
-        .then((answer) => {
-          if (answer === "Yes") {
-            this.set(toolchainEnv, toolchain);
-            return resolve(true);
-          } else {
-            return reject(null);
-          }
-        });
-    });
-  }
-
   public set(toolchainEnv: ToolchainEnv, toolchain: Toolchain) {
     if (this.isEqual(toolchain)) {
       return;
     }
     this._toolchainEnv = toolchainEnv;
     this._toolchain = toolchain;
```

Comment:
If you're gonna remove the information message, how about replacing it with balloon or logger(console)?

Reply:
Updated this patch. Please see the latest code.